tutorials/LinearAlgebra/LinearAlgebra.ipynb

```diff
@@ -0,0 +1,983 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to Linear Algebra\n",
+    "\n",
+    "This is a tutorial designed to introduce you to the basics of linear algebra.\n",
+    "Linear algebra is a branch of mathematics dedicated to studying the properties of matrices and vectors.\n",
+    "These are used extensively in quantum computing to represent quantum states and operations on them.\n",
+    "While this tutorial doesn't even come close to covering the full breadth of the topic, it should be enough to get you started.\n",
+    "In particular, you should be comfortable with the linear algebra used in quantum computing.\n",
+    "\n",
+    "It is highly recommended that you complete the [Complex Arithmetic](../ComplexArithmetic/ComplexArithmetic.ipynb) tutorial before tackling this one.\n",
+    "\n",
+    "This tutorial covers the following topics:\n",
+    "* Matrices and vectors\n",
+    "* Basic matrix operations\n",
+    "* Inner product\n",
+    "* Outer product\n",
+    "* Tensor product\n",
+    "* Eigenvalues and eigenvectors\n",
+    "* Complex matrices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook has several tasks that require you to write Python code to test your understanding of the concepts. If you are not familiar with Python, [here](https://docs.python.org/3/tutorial/index.html) is a good introductory tutorial for it.\n",
+    "\n",
+    "The exercises use Python's built-in representation of complex numbers. Most of the operations (addition, multiplication, etc.) work as you expect them to, but there are a few Python-specific caveats:\n",
```

Comment:
I would make the section about complex numbers (from this sentence up to "You can find more information...") a note (marked with `>`) to split the wall of text into several sections.